### PR TITLE
Simple spec refactoring to extract fixture path helpers

### PR DESCRIPTION
**Pull request by joehorsnell**
I noticed while doing another PR (to be opened shortly!) that there is a bit of duplication in the specs around generating fixture paths, so this simple refactoring just extracts a `fixture_path` method to DRY it up.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -2,11 +2,19 @@
 require "coveralls"
 
 SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-SimpleCov.start 
+SimpleCov.start
 
 require File.dirname(__FILE__) + '/../lib/require_all.rb'
 
 module SpecHelper
+  def fixture_path(fixture_name, relative_dir = File.dirname(__FILE__))
```
Comment (owst) [inline spec/spec_helper.rb]: Small refactoring: `File.dirname(__FILE__)` can be more succinctly written as `__dir__`

Comment (jarmo) [reply spec/spec_helper.rb]: If it passes on CI, then there's no problem. I'm pretty sure it will pass there with this change too.

**Comment by joehorsnell** [comment]
Thoughts on this one @jarmo? If you merge #22 I can then make the change suggested above by @owst.

**Comment by jarmo** [comment]
Please merge conflicts and I'll merge this in.

**Comment by joehorsnell** [comment]
Done @jarmo

**Comment by jarmo** [comment]
Merged. Thanks!

**Pull request merged**